### ISSUE TITLE
ARFO won't load, every time I try to start MC it gives me error messages like this one (crash txts attached)

### ISSUE THREAD

**Issue by ghost** [finallion/Artificial_Foliage-FORGE-#3]
![arfo](https://user-images.githubusercontent.com/81043981/144937594-f9748116-8441-45f3-8be6-c77befde676d.png)
[crash-2021-12-06_17.43.40-fml.txt](https://github.com/finallion/Artificial_Foliage-FORGE-/files/7664129/crash-2021-12-06_17.43.40-fml.txt)
[latest log.txt](https://github.com/finallion/Artificial_Foliage-FORGE-/files/7664130/latest.log.txt)

**Comment by finallion** [comment]
Could you post your logs, and which version of forge and the mod itself are you using?

**Comment by ghost** [comment]
I thought I did, that's what the "latest log.txt" thing is supposed to be right? or is it some other log?
forge version is "forge-1.17.1-37.0.104-installer"
mod version is "Artificial_Foliage_1.0_(FORGE)_for_1.17.1"

**Comment by finallion** [comment]
I'll look into into it.
Sorry - didn't notice the attached logs!
Did you try using a lower Forge version?

**Comment by ghost** [comment]
No, not yet. I suppose I could try an older version but it would probably interfere with my other mods lol
